vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "probeVmSize": {
```

Comment:
But what if the user has a specific instance type in mind and wants to validate real quick? For (A-series networking is different enough, for example) The idea was to give a tool not just the example.. I understand this makes more lines, however. Failing for reason template is also educating a user, if the link in README is ignored.